File: solutions/java/resistor-color/1/src/main/java/ResistorColor.java
import java.util.Arrays;

class ResistorColor {
    static private String[] COLORS = {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"};
    
    static int colorCode(String color) {
        return Arrays.asList(COLORS).indexOf(color);
    }

    String[] colors() {
        return Arrays.copyOf(COLORS, COLORS.length);
    }
}

File: solutions/kotlin/resistor-color/1/src/main/kotlin/ResistorColor.kt
object ResistorColor {
    val COLORS = listOf("black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white")

    fun colorCode(input: String): Int =
        COLORS.indexOf(input)

    fun colors(): List<String> =
        COLORS
}
